packages/@o3r/testing/schematics/ng-add/fixture/index.ts

```diff
@@ -59,13 +59,24 @@ export function updateFixtureConfig(options: { projectName: string | null | unde
         }
         return acc;
       }, {});
-      const subFolder = options.testingFramework === 'jasmine' ? 'karma' : 'angular';
-      tsconfigCompilerOptions.paths['@o3r/testing/core'] = [`node_modules/@o3r/testing/core/${subFolder}`];
-      tsconfigCompilerOptions.paths['@o3r/testing/core/*'] = [`node_modules/@o3r/testing/core/${subFolder}/*`];
+
+      const testFramework = options.testingFramework || getTestFramework(getWorkspaceConfig(tree)!);
+
+      let installJest = false;
```

Comment:
I don't think the `if (testFramework !== 'other')` is mandatory here
```typescript
const installJest = (
  testFramework === 'jest'
  || (
     testFramework === undefined
     && await askConfirmation('Do you want to setup Jest test framework?', true)
  )
);

Reply:
if `other` is passed as inline param or by another schematics we want to make sure it takes priority.

Reply:
if `other` is given through inline param or by another schematic
`testFramework` will not be `undefined` neither `jest` so it will still not ask the question
the check `testFramework !== 'other'` doesn't add any check in that case no ?
because we already check `jest` and `undefined`